approaches/transformers.md

```diff
@@ -1,7 +1,41 @@
 ---
 grand_parent: Approaches
 parent: Neural machine translation
-layout: coming_soon
 title: Transformers
 description: A deep learning model architecture for neural machine translation
 ---
+
+**Transformers** are a type of neural network architecture that relies only on the [attention](/concepts/attention.md) mechanism.
+It allows [neural machine translation](/approaches/neural-machine-translation.md) models to better handle the dependencies between the input and output sequences.
```

Comment:
Should we keep with the plural of `transformers` and say "They allow..."?
Or is it better to turn `transformers` into `tranformer`?

Reply:
I believe it's better to continue with the singular form as the original paper on this subject calls it `the Transformer architecture, with `Transformer` being the main name for the model architecture. That's why I also wrote that it's a type of neural network architecture, using singular `relies` to address the type (which you mentioned in the previous comment).

Reply:
Ah perfect! Let's switch to the singular then. Should we rename the file?

Reply:
That's a good idea. I'll make the changes now.

Reply:
> (which you mentioned in the previous comment)

I deleted it because I had interpreted it incorrectly.
I thought it was "Transformers are a type of neural network architecture that rely [the transformers]", but now I understand it's actually "Transformers are a type of neural network architecture that relies [the architecture]"...
So, my bad! Sorry about that one.

Reply:
Also, I've thought about renaming the file to `Transformer`, and I think it's best to keep it in the plural form as there can be different types of transformers with slightly changed architecture but the same idea.